gm4_combat_expanded/data/gm4_combat_expanded/functions/armor/modifier/type/canine/wolf_checked.mcfunction

```diff
@@ -3,5 +3,5 @@
 # at world spawn
 # run from armor/active/canine/check
 
+execute if score $wolf gm4_ce_data matches 0 run scoreboard players set @s gm4_ce_t_wolf 1
```

Comment:
`$wolf gm4_ce_data` is always 0 when this function is called from `armor/type/canine/check`

Reply:
Technically $wolf can change to 1 if the player has multiple dogs, but it's not really worth checking the score every time. It's actually more efficient to just set the score since it doesn't have to check anything; just has to overwrite.

Reply:
the idea behind checking if `$wolf` is 0 is that if you have multiple wolves linked to this armor piece only one will actually get the `gm4_ce_t_wolf` score and remain. Otherwise unloading a wolf and then walking back leaves you with a duplicated one

Reply:
Oh, I got confused on the context; I thought this was ran as the owner, not the dog. The change is good then